etc/ci-gather-coverage.sh

```diff
@@ -71,7 +71,40 @@ fi;
 # Always output coveralls JSON, as ci-coveralls-merge.py relies
 # on it being present and containing all relevant metadata...
 Print("Outputting JSON for Coveralls...\n");
-OutputCoverallsJsonCoverage(r, "gap-coveralls.json", prefix);
+
+env := GAPInfo.SystemEnvironment;;
+if IsBound(env.GITHUB_ACTIONS) then
+    opt := rec(
+        service_name := "github",
+        # The build number. Will default to chronological numbering from builds on repo
+        service_number := env.GITHUB_RUN_NUMBER,
+        # A unique identifier of the job on the service specified by service_name
+        # FIXME: there seems to be no unique per-job id; the GITHUB_RUN_ID is
+        # shared by all jobs in a single build :-(
+        #service_job_id := env.GITHUB_RUN_ID,
+        service_branch := env.GITHUB_REF{[Length("refs/heads/")..Length(env.GITHUB_REF)]},
```

Comment:
So even with https://github.com/marketplace/actions/coveralls-github-action we need to invest work to handle parallel jobs (= the default for almost everybody using GitHub Action Workflows, sheesh, thanks for making everybody work for that Coveralls :-( and kudos to Codecov for making do without.

I think the only thing we can really do here is to manually assign a "name" to each of our CI jobs, which we can then use. Actually, perhaps we can just use the existing `name: ${{ matrix.test-suites }} - ${{ matrix.extra }} - ${{ matrix.os }}` field for that.